common/reviews/api/heft-storybook-plugin.api.md

```diff
@@ -12,9 +12,10 @@ export default _default;
 
 // @public
 export interface IStorybookPluginOptions {
-    startupModulePath?: string;
-    staticBuildModulePath?: string;
-    storykitPackageName: string;
+  options?: string[];
```

Comment:
I prefer name it `cliOptions` instead

Reply:
I think making the options provided to Heft as explicit as possible is the best route IMO... we could use an option named `outputPath` and only append the applicable parameter to the `process.argv`